modules/rosa-upgrading-cli-tutorial.adoc

```diff
@@ -32,9 +48,13 @@ The command returns a list of versions to which the the cluster can be upgraded,
 +
 [source,terminal]
 ----
-$ rosa upgrade cluster --cluster=<cluster_name>
+$ rosa upgrade cluster --cluster=<cluster_name|cluster_id>
 ----
 +
 The cluster is scheduled for an immediate upgrade. This action can take an hour or longer, depending on your workload configuration, such as PodDisruptionBudgets.
```

Comment:
Not part of your update, but please nab pod disruption budget when the opportunity presents itself.

Reply:
Good catch. I have resolved this in the PR.